resources/application.conf

```diff
@@ -11,5 +11,6 @@ ktor {
 wallet {
     alreadySynced = false
     apiPassword = "password"
-    mnemonic = "test test test test test test test test test test test test"
+    descriptor = "wpkh(tprv8ZgxMBicQKsPeqkztELFj4aDR1P3U2fc8k56wgujCurY8pmKDPGN5RKjF1ENNWxcWygdFuiWzBWT1zP2AQcSjZC3u9UfQjciisdLwL58kdk/84'/1'/0'/0/*)"
+    change_descriptor = "wpkh(tprv8ZgxMBicQKsPeqkztELFj4aDR1P3U2fc8k56wgujCurY8pmKDPGN5RKjF1ENNWxcWygdFuiWzBWT1zP2AQcSjZC3u9UfQjciisdLwL58kdk/1/*)"
```

Comment:
Your change descriptor is simply the same `tprv` but with a different path (you can take a look at [this article](https://thunderbiscuit.com/modern-wallets/)).

The proper change descriptor in this case would be:
```kotlin
wpkh(tprv8ZgxMBicQKsPeqkztELFj4aDR1P3U2fc8k56wgujCurY8pmKDPGN5RKjF1ENNWxcWygdFuiWzBWT1zP2AQcSjZC3u9UfQjciisdLwL58kdk/84'/1'/0'/1/*)
```